configure.d/1_bdev_release.conf

```diff
@@ -40,6 +43,9 @@ apply() {
 		add_define "cas_bdev_release(handle, mode, holder) \\
 			bdev_release(handle)" ;;
 	"4")
+		add_define "cas_bdev_release(handle, mode, holder) \\
+			__fput_sync(handle)" ;;
```

Comment:
`fput()` schedules a delayed work to release the files, so it's just enough to call `reschedule()` just after `fput()` to let the delayed work run.

Reply:
I've just tested `({ fput(handle); schedule(); })"` and it work (kernel 5.14.0-479.el9.x86_64)